exporter/datadogexporter/integrationtest/integration_test.go

```diff
@@ -414,3 +438,89 @@ func sendTracesComputeTopLevelBySpanKind(t *testing.T) {
 	}
 	time.Sleep(1 * time.Second)
 }
+
+func TestIntegrationLogs(t *testing.T) {
+	// 1. Set up mock Datadog server
+	// See also https://github.com/DataDog/datadog-agent/blob/49c16e0d4deab396626238fa1d572b684475a53f/cmd/trace-agent/test/backend.go
+	seriesRec := &testutil.HTTPRequestRecorderWithChan{Pattern: testutil.MetricV2Endpoint, ReqChan: make(chan []byte)}
+	doneChannel := make(chan bool)
+	var connectivityCheck sync.Once
+	var logsData testutil.JSONLogs
+	server := testutil.DatadogLogServerMock(seriesRec.HandlerFunc, func() (string, http.HandlerFunc) {
+		return "/api/v2/logs", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
nit: can this http handler be part of `DatadogLogServerMock`? (to avoid adding this handler in different tests over and over again)

Reply:
Will address this in a separate PR